src/api/app/models/issue_tracker.rb

```diff
@@ -100,6 +100,11 @@ def update_issues_github
     mtime = Time.now
 
     response = follow_redirects(url)
+
+    if response.code != '200'
+      raise "IssueTracker ##{id} could not connect to github.\nUrl: #{url}\nResponse: #{response.body}"
```

Comment:
I'm not so sure we need to raise an exception here. I would rather log this and continue..